src/main/java/core/basesyntax/figuresupplier/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figuresupplier;
+
+import core.basesyntax.Figure;
+import core.basesyntax.colorsupplier.ColorSupplier;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String COLOR = "WHITE";
+    public static final int RADIUS = 10;
+    private static final Random random = new Random();
+    private double firstLine;
+    private double secondLine;
+    private double thirdLine;
+    private ColorSupplier colorSupplier;
+
+    public Figure getDefaultFigure() {
+        return new Circle(COLOR, RADIUS);
+    }
+
+    public Figure getRandomFigure() {
+        firstLine = new Random().nextInt(50);
+        secondLine = new Random().nextInt(50);
+        thirdLine = new Random().nextInt(50);
+        int rundomFigura = random.nextInt(6);
+        colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = "WHITE";
    private static final int DEFAULT_RADIUS = 10;
    private static final int FIGURE_NUMBER = 6;
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();

    public Figure getDefaultFigure() {
        return new Circle(COLOR, RADIUS);
    }

    public Figure getRandomFigure() {
        int rundomFigura = random.nextInt(FIGURE_NUMBER);
```
firstLine, secondLine, thirdLine - local variables